CONTRIBUTING.md

```diff
@@ -67,6 +67,9 @@ don't see it, feel free to [open a new issue][open an issue].
 
 [open an issue]: https://github.com/badges/shields/issues/new
 
+### Starring the repository
+
+Feel free to star the repository. This will help increase the visibility of the project, therefore attracting more users and contributors to Shields! 
```

Comment:
Great idea to include this! I’m cool with it as is. Another idea: do you think we should expand the call to action to “Spread the word,” and include asking people to share that we ask developers who to use and love shields to give $10.

Reply:
Done, how does it look?

Reply:
Woot!